src/ClientData/include/ClientData/TextBox.h

```diff
@@ -27,31 +24,32 @@ class TextBox {
   TextBox(TextBox&& other) = default;
   TextBox& operator=(TextBox&& other) = delete;
 
-  void SetSize(const Vec2& size) { size_ = size; }
-  void SetPos(const Vec2& pos) { pos_ = pos; }
+  void SetSize(std::pair<float, float> size) { size_ = std::move(size); }
+  void SetPos(std::pair<float, float> pos) { pos_ = std::move(pos); }
 
-  const Vec2& GetSize() const { return size_; }
-  const Vec2& GetPos() const { return pos_; }
+  [[nodiscard]] const std::pair<float, float>& GetSize() const { return size_; }
+  [[nodiscard]] const std::pair<float, float>& GetPos() const { return pos_; }
 
-  const std::string& GetText() const { return text_; }
+  [[nodiscard]] const std::string& GetText() const { return text_; }
   void SetText(std::string text) { text_ = std::move(text); }
 
   const orbit_client_protos::TimerInfo& GetTimerInfo() const { return timer_info_; }
```

Comment:
[[nodiscard]], I you are already updating those.

Reply:
done.